packages/stepper/src/Stepper.tsx

```diff
@@ -0,0 +1,521 @@
+  justify-content: center;
+  height: 100%;
+  width: 100%;
+  margin-left: -${46 / 3 + 1}px;
+
+  color: ${uiColors.gray.dark3};
+
+  font-family: Akzidenz, 'Helvetica Neue', Helvetica, Arial, sans-serif;
+  font-size: 14px;
+  font-weight: bold;
+
+  z-index: 2;
+
+  &:first-of-type {
+    .${stepBoxTopStyle}, .${stepBoxBottomStyle} {
+      &:before {
+        transform: none;
+      }
+    }
+  }
+
+  &:last-of-type {
+    .${stepBoxTopStyle}, .${stepBoxBottomStyle} {
+      &:before,
+      &:after {
+        transform: none;
+      }
+    }
+  }
+`;
+
+const currentStepStyle = css`
+  filter: drop-shadow(0px 4px 4px ${transparentize(0.7, uiColors.black)});
+  z-index: 3;
+
+  &:last-of-type:not(:first-of-type) {
+    .${stepBoxTopStyle}:before {
+      transform: skewX(${Math.atan(2 / 3)}rad);
+    }
+
+    .${stepBoxBottomStyle}:before {
+      transform: skewX(-${Math.atan(2 / 3)}rad);
+    }
+  }
+`;
+
+const upcomingStepStyle = css`
+  color: ${uiColors.gray.base};
+  z-index: 1;
+
+  &:last-of-type {
+    z-index: 0;
+  }
+
+  .${stepBoxTopStyle}, .${stepBoxBottomStyle} {
+    & > *,
+    &:before,
+    &:after {
+      background: ${uiColors.gray.light3};
+      border-color: ${uiColors.gray.light2};
+    }
+  }
+`;
+
+const hiddenLayerStyle = css`
+  opacity: 0;
+  visibility: hidden;
+  pointer-events: none;
+`;
+
+const slideInAnimationStyle = css`
+  left: 0;
+  transition: left 700ms;
+`;
+
+const slideOutAnimationStyle = css`
+  left: -30px;
+  transition: left 700ms;
+`;
+
+const containerStyle = css`
+  position: relative;
+  height: 46px;
+  width: 100%;
+`;
+
+const stepLabelStyle = css`
+  display: flex;
+  position: absolute;
+
+  align-items: center;
+  justify-content: center;
+  top: calc(100% - 12px);
+
+  height: 24px;
+  min-width: 24px;
+  max-width: max-content;
+
+  font-size: 12px;
+  line-height: 14px;
+
+  border-radius: 12px;
+  border: 1px solid;
+`;
+
+const completedStepLabelStyle = css`
+  background-color: white;
+  border-color: ${uiColors.green.base};
+  color: ${uiColors.green.base};
+`;
+
+const currentStepLabelStyle = css`
+  background-color: ${uiColors.gray.dark2};
+  border-color: #464c4f;
+  color: white;
+`;
+
+const upcomingStepLabelStyle = css`
+  background-color: white;
+  border-color: ${uiColors.gray.light1};
+  color: ${uiColors.gray.base};
+`;
+
+const stepLabelTextStyle = css('padding: 0 8px;');
+
+function InternalStep({
+  children,
+  state,
+  stepLabel,
+  className,
+  ...containerProps
+}: InternalStepProps) {
+  const isCurrent = state === 'current';
+  const isUpcoming = state === 'upcoming';
+  const isCompleted = state === 'completed';
+  return (
+    <div
+      className={cx(
+        stepStyle,
+        { [upcomingStepStyle]: isUpcoming },
+        { [currentStepStyle]: isCurrent },
+        className,
+      )}
+      {...containerProps}
+    >
+      {stepLabel && (
+        <div
+          className={cx(
+            stepLabelStyle,
+            state === 'current'
```

Comment:
Might be cleaner to use `isCurrent` here since you've already created the variable

Reply:
I also think we should be consistent, either conditionally applying styles with the object notation as you did above, or through a nested ternary as you're doing here.